lib/cext/preprocess.rb

```diff
@@ -270,7 +270,7 @@ def read_write_field(struct_var_name, field_name, leaking)
     gem: 'byebug',
     patches: [
       {
-        match: /\b(static VALUE \w+) = (?:Qfalse|Qnil);/,
+        match: /\b((?:static )?VALUE \w+) = (?:Qfalse|Qnil);/,
```

Comment:
Should there be an extra `^` anchor instead of just `\b`?
Otherwise it seems this could apply to local variable which would be bad.

Reply:
Fixed, anchored.